src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -314,31 +214,28 @@ def __init__(self, config: BertConfig, **kwargs):
                 f"of attention heads ({config.num_attention_heads})"
             )
 
+        self.num_attention_heads = config.num_attention_heads
         self.attention_head_size = int(config.hidden_size / config.num_attention_heads)
+        self.all_head_size = self.num_attention_heads * self.attention_head_size
+        self.rsqrt_att_head_size = 1.0 / math.sqrt(self.attention_head_size)
```

Comment:
Not sure if @mfuntowicz shared it with you, but he observed some instability when using this version, whereas sorting the `sqrt` (just `math.sqrt(self.attention_head_size)`) and then using `tf.divide` in the call instead of `tf.multiply` was more stable.

Reply:
Ok, let's put this instead then :)

Reply:
Ok, now it should be ok!